firstvoices/backend/management/commands/convert_draftjs_to_html.py

```diff
@@ -0,0 +1,102 @@
+import json
+import logging
+
+from django.core.management.base import BaseCommand
+from draftjs_exporter.html import HTML
+
+from backend.models import Site, Song, Story, StoryPage
+from backend.models.widget import SiteWidget, WidgetSettings
+
+
+class Command(BaseCommand):
+    help = "Convert fv-be draftjs content to html content."
+
+    def add_arguments(self, parser):
+        parser.add_argument(
+            "--sites",
+            dest="site_slugs",
+            help="Site slugs of the sites to convert draftjs content for, separated by comma (optional)",
+            default=None,
+        )
+
+    @staticmethod
+    def is_draftjs_content(text):
+        try:
+            content = json.loads(text)
+            return (
+                isinstance(content, dict)
+                and "blocks" in content
+                and "entityMap" in content
+            )
+        except json.JSONDecodeError:
+            return False
+
+    def convert_draftjs_to_html(self, text):
+        if self.is_draftjs_content(text):
+            exporter = HTML()
+            return exporter.render(json.loads(text))
+        return text
+
+    def handle(self, *args, **options):
+        logger = logging.getLogger("convert_draftjs_to_html")
+        logger.setLevel(logging.INFO)
+
+        if options.get("site_slugs"):
+            site_slug_list = [
+                s.strip() for s in options.get("site_slugs", "").split(",")
+            ]
+            sites = Site.objects.filter(slug__in=site_slug_list)
+            if not sites:
+                logger.info("No sites with the provided slug(s) found.")
+                return
+        else:
+            sites = Site.objects.all()
+
+        for site in sites:
+            logger.info(f"Converting draftjs content to html for site {site.slug}...")
```

Comment:
Looking at this and the rest of the logs in this method-- I think they should all be debug level logs